packages/plugins/plugin-table/src/components/TableContainer.tsx

```diff
@@ -107,11 +104,7 @@ const TableContainer = ({ role, table }: LayoutContainerProps<{ table: TableType
 
   return (
     <StackItem.Content toolbar role={role}>
-      <Toolbar.Root onAction={handleAction} classNames={!hasAttention && 'opacity-20'}>
-        <Toolbar.Editing />
-        <Toolbar.Separator />
-        <Toolbar.Actions viewDirty={model?.isViewDirty} />
-      </Toolbar.Root>
+      <TableToolbar onAction={handleAction} classNames={['p-1 attention-surface', !hasAttention && 'opacity-20']} />
```

Comment:
thinking that these class names should be factored out to the intermediate component since we will want all our toolbars to have this behaviour. Attention is one of the few context's which also totally makes sense for it to not `raise` when requesting it. If the context is not available then naturally attention is false so it works fine at lower levels without adding the overhead of needing an attention provider in every story.

Reply:
My first impression is this feels like not quite the right separation of concerns, this would mean react-ui-menu should depend on react-ui-attention. I’m amenable to trying it, though.

Reply:
yeah, I think that's okay, I'd argue our intermediate components should be attention-aware where it makes sense if there's an attention provider in the tree

Reply:
I can appreciate this essentially means that attention as a concept is upstream of even intermediate components, sitting between low-level UI (react-ui) and those, that does seem reasonable.